caikit/runtime/service_generation/data_stream_source.py

```diff
@@ -241,13 +238,15 @@ def make_data_stream_source(data_element_type: Type) -> Type[DataBase]:
                 File.__name__: File,
                 ListOfFiles.__name__: ListOfFiles,
                 Directory.__name__: Directory,
+                S3Files.__name__: S3Files,
```

Comment:
Any reason you didn't also include `S3Path`?

Reply:
We thought `S3Path` would be useful for the `output_path` parameter where you only want to allow one path to be passed. But for an input stream, a user should be free to specify multiple.

I'm fine either way if we want to add both here similar to `File` and `ListOfFiles`